LayoutTests/css3/filters/effect-blur-hw.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <meta name="fuzzy" content="maxDifference=0-54; totalPixels=0-61800">
+    <meta name="fuzzy" content="maxDifference=0-99; totalPixels=0-61800">
```

Comment:
Not new in this PR, but do these tests actually verify anything for real? With so much fuzziness, it seems like the actual results could bear little semblance to the expectation.